client/layout/masterbar/logged-in.jsx

```diff
@@ -207,38 +207,41 @@ class MasterbarLoggedIn extends Component {
 		return 'my-sites';
 	};
 
-	// will render as back button on mobile and in editor
-	renderMySites() {
-		const {
-			domainOnlySite,
-			hasMoreThanOneSite,
-			siteSlug,
-			translate,
-			isCustomerHomeEnabled,
-			section,
-		} = this.props;
-		const { isMenuOpen, isResponsiveMenu } = this.state;
+	renderWordPressIcon() {
+		const { siteSlug, translate, isCustomerHomeEnabled } = this.props;
+		const { isMenuOpen } = this.state;
 
 		const homeUrl = isCustomerHomeEnabled
 			? `/home/${ siteSlug }`
 			: getStatsPathForTab( 'day', siteSlug );
 
-		let mySitesUrl = domainOnlySite ? domainManagementList( siteSlug ) : homeUrl;
-		if ( 'sites' === section && isResponsiveMenu ) {
-			mySitesUrl = '';
-		}
-		const icon =
-			this.state.isMobile && this.props.isInEditor ? 'chevron-left' : this.wordpressIcon();
 		return (
 			<Item
-				url={ mySitesUrl }
+				url={ homeUrl }
+				icon={ this.wordpressIcon() }
+				isActive={ this.isActive( 'sites' ) && ! isMenuOpen }
+				tooltip={ translate( 'View a list of your sites and access their dashboards' ) }
+			/>
+		);
+	}
+
+	// will render as back button on mobile and in editor
+	renderMySites() {
+		const { hasMoreThanOneSite, translate } = this.props;
+		const { isMenuOpen } = this.state;
+
+		//TODO make sure correct WordPress icon is used on Horizon sites
+		const icon = this.state.isMobile && this.props.isInEditor ? chevronLeft : layout;
+		return (
+			<Item
+				url="/sites"
 				tipTarget="my-sites"
-				icon={ icon }
 				onClick={ this.clickMySites }
 				isActive={ this.isActive( 'sites' ) && ! isMenuOpen }
```

Comment:
The active state uses the old section name.

```suggestion
				isActive={ this.isActive( 'sites-dashboard' ) && ! isMenuOpen }
```



**/sites** screenshot:
| **Before** | **After** |
|---|---|
| ![before](https://user-images.githubusercontent.com/779993/205631961-6d17e6a4-5cf5-4fcf-9e49-b0eaf88ad787.png) | ![after](https://user-images.githubusercontent.com/779993/205632103-e12e591f-ea53-4a87-aa75-de793a50f01f.png) |

**/home/site** screenshot:
| **Before** | **After** |
|---|---|
| <img width="272" alt="Screenshot 2022-12-05 at 12 08 22" src="https://user-images.githubusercontent.com/779993/205633824-63cefa0c-fd70-4450-8c4b-4b67bd5c5256.png"> | <img width="273" alt="Screenshot 2022-12-05 at 12 07 39" src="https://user-images.githubusercontent.com/779993/205633841-aca367d4-d6ea-4da0-b9fe-a95a2bca2d36.png"> |